etc/fixtures/add_user.js

```diff
@@ -0,0 +1,75 @@
+const usage = "\n\nmongosh --eval \"var help=false; [options]\" /tmp/fixtures/add_user.js\n" +
```

Comment:
Some thing has that been found is that `mongosh` has the ability to accept and interpret both the host and port variables without needing to pass them in as arguments to the node script. 

The database name might be able to be passed in as well, but it is okay to leave it as a node script arg.

Please see for details:

[https://www.mongodb.com/docs/mongodb-shell/write-scripts/#script-does-not-include-connection-details](https://www.mongodb.com/docs/mongodb-shell/write-scripts/#script-does-not-include-connection-details)